### ISSUE TITLE
Replace Leo's video with Santiago's KEGS talk

### ISSUE THREAD

**Issue by leouieda** [fatiando/website#83]
@santisoler recently gave a recorded talk on Fatiando at KEGS. It's a better introduction and overview than the video we currently use in the front page and "About". Once it's up on our YouTube, use that instead.

**Comment by aguspesce** [comment]
Hello @leouieda 
Taking about that... Maybe we can add a tab or a section or update a section with the talks and videos where fationdo was presented.

**Comment by leouieda** [comment]
> Maybe we can add a tab/section or update a section with the talks and videos where fatiando was presented.

Yes! How about we change the [About page](https://www.fatiando.org/about/) to have:

1. Talks about Fatiando. List them and include the youtube preview and link to the playlist that is currently there.
1. The geophysics Python ecosystem
1. Brief history of the project

What do you think?

**Comment by aguspesce** [comment]
@leouieda I like the idea. I will tray to start with this next week
